content/news/2022-01-06-bevy-0.6/index.md

```diff
@@ -0,0 +1,1007 @@
+
+This benchmark spawns entities with a variety of component compositions to ensure we cover a variety of cases. Treat these numbers as realtives, not absolutes.
+
+![commands perf](commands_perf.svg)
+
+### System Param Lifetimes
+
+<div class="release-feature-authors">authors: @cart, @BoxyUwU, @TheRawMeatball</div>
+
+System and Query lifetimes were made more explicit by splitting out the `'system` and `'world`, lifetimes and using them explicitly where possible. This enables Rust to reason about ECS lifetimes more effectively, especially for read-only lifetimes. This was particularly important because it enabled the new Bevy Renderer to convince WGPU that ECS resources actually live for as long as the Render World.
+
+### Soundness / Correctness Improvements
+
+<div class="release-feature-authors">authors: @BoxyUwU, @TheRawMeatball, @Frizi, @thebluefish, @sapir, @bjorn3, @DJMcNab</div>
+
+Bevy ECS received a solid number of soundness and correctness improvements this release, alongside some unsafe code removals. Queries and internal storages like Tables and BlobVecs in particular had a number of improvements in these areas. And we have even more improvements in the works!
+
+### Hierarchy Convenience Functions
+
+<div class="release-feature-authors">authors: @TheRawMeatball, @jihiggins</div>
+
+```rust
+// Despawns all descendants of an entity (its children, its childrens' children, etc)
+commands.entity(e).despawn_descendants();
+
+// Removes the given children from the entity
+commands.entity(parent).remove_children(&[child1, child2]);
+```
+
+## UI
+
+### Overflow::Hidden
+
+<div class="release-feature-authors">authors: @Davier</div>
+
+UI now repects the flexbox `Overflow::Hidden` property. This can be used to cut off child content, which is useful when building things like scrollable lists:
+
+![overflow hidden](overflow.png)
+
+### Text2D Transforms
+
+<div class="release-feature-authors">authors: @nside</div>
+
+`Text2d` now supports arbirtary transformations using the Transform component:`
+
+![text 2d transforms](text2d_transforms.png)
+
+Note that while `Transform::scale` does have it uses, it is generally still a good idea to adjust text size using the "font size" to ensure it renders "crisply".
+
+## Profiling
+
+### More Spans
+
+<div class="release-feature-authors">authors: @cart, @mockersf, @hymm</div>
+
+The new renderer now has tracing spans for frames, the render app schedule, and the Render Graph (with named Sub Graphs spans). The system executor now has finer grained spans, filling in most of the remaining blanks. Applying System Commands also now has spans.
+
+(ignore those weird characters in the spans ... we're investigating that)
+
+![render app profiling](render_app_profiling.png)
+
+### Tracy Backend
+
+<div class="release-feature-authors">authors: Rob Swain (@superdump)</div>
+
+We now have built in support for the [tracy](https://github.com/wolfpld/tracy) profiler via the `trace_tracy` Cargo feature.
```

Comment:
Maybe a screenshot here too? I think I had a good one showing the stats of system execution time for cluster light assignment, or frustum culling.